test_kcidb.py

```diff
@@ -32,6 +34,30 @@ def test_json_output_options():
                     stdout_re="\\{\n  \".*")
 
 
+def test_package_symbols():
+    """
+    Test that package modules are accessible as symbols after importing
+    the package's module (__init__.py).
+
+    Starting with the kcidb package, this test lists all modules under
+    it using pkgutil, imports the package and verifies that each listed
+    module is available as a symbol with type <class 'module'> inside it.
+    It then continues to all sub-packages recursively until everything
+    is verified.
```

Comment:
I think the second paragraph here is redundant and describes the algorithm, which is better discovered by reading the actual (very short) code. Could you delete it?

Reply:
Okay